doc/v2.rst

```diff
@@ -0,0 +1,86 @@
+##################################
+Use v2 of the SageMaker Python SDK
```

Comment:
```suggestion
Use v2 of the SageMaker Python SDK (experimental)
```
Out of context, and as a second top node, you'll get a lot of clicks, but folks may not be ready to go for experimental options.